x/stakeibc/keeper/icacallbacks_rebalance.go

```diff
@@ -42,7 +41,7 @@ func RebalanceCallback(k Keeper, ctx sdk.Context, packet channeltypes.Packet, ac
 	txMsgData, err := icacallbacks.GetTxMsgData(ctx, *ack, k.Logger(ctx))
 	if err != nil {
 		k.Logger(ctx).Error(fmt.Sprintf("failed to fetch txMsgData, packet %v", packet))
-		return sdkerrors.Wrap(icacallbackstypes.ErrTxMsgData, err.Error())
+		return fmt.Errorf(icacallbackstypes.ErrTxMsgData.Error(), err.Error())
```

Comment:
@hoangdv2429 I think it should be `return fmt.Errorf("%s: %s", err.Error(), icacallbackstypes.ErrTxMsgData.Error())`